ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/utils.ts

```diff
@@ -81,17 +80,7 @@ export function getReportConfigurationFromFormValues(
         };
     }
 
-    const notifiers = deliveryDestinations.map((deliveryDestination) => {
-        return {
-            emailConfig: {
-                notifierId: deliveryDestination.notifier?.id || '',
-                mailingLists: deliveryDestination.mailingLists,
-                customSubject: deliveryDestination.customSubject,
-                customBody: deliveryDestination.customBody,
-            },
-            notifierName: deliveryDestination.notifier?.name || '',
-        };
-    });
+    const notifiers = deliveryDestinations;
```

Comment:
😁

Reply:
Yeah, besides reducing the number of changes, `notifiers` as name of property in payload and response is ambiguous with plain `notifiers` from GET /v1/notifiers request.